config/default.json

```diff
@@ -5,7 +5,7 @@
     "atnaUrl": "mongodb://localhost/openhim",
     "connectionOptions": {},
     "openHIMApiReadPreference": "primaryPreffered",
-    "openHIMApiWriteConcern": "1",
+    "openHIMApiWriteConcern": "majority",
```

Comment:
I initially missed this as well:
https://docs.mongodb.com/manual/reference/write-concern/